server/src/authorizers/permissions.js

```diff
@@ -3,24 +3,36 @@ const { shield, allow, and, or, not } = require('graphql-shield');
 
 const { isAuthenticated } = require('./agents');
 const { isChannelPublic } = require('./channels');
-const { isAdmin, isViewer, isInChannel } = require('./channel-memberships');
-const { isMoreThenOneAdmin } = require('./channel-memberships');
-const { isSubjectAsObject } = require('./channel-memberships');
-const { isObjectAdmin } = require('./channel-memberships');
-const measurables = require('./measurables');
-const bots = require('./bots');
+const { isAdmin, isViewer } = require('./channel-memberships');
+const { channelHasMembershipWithCurrentAgent } = require('./channel-memberships');
+const { channelHasMultipleAdmins } = require('./channel-memberships');
+const { membershipBelongsToCurrentAgent } = require('./channel-memberships');
+const { membershipHasAdminRole } = require('./channel-memberships');
+const { measurableIsOwnedByCurrentAgent } = require('./measurables');
+const { measurableIsArchived } = require('./measurables');
+const { botBelongsToCurrentUser } = require('./bots');
 
 const rulesChannel = {
   Query: {},
   Mutation: {
-    channelUpdate: and(isAuthenticated, isAdmin),
+    channelUpdate: and(
+      isAuthenticated,
+      isAdmin,
```

Comment:
can we change ``isAdmin`` to ``currentUserIsAdmin``?

Reply:
Updated.